src/components/Settings/SettingsDialog.tsx

```diff
@@ -72,9 +68,7 @@ export const SettingsDialog = ({ open, onOpenChange }: SettingsDialogProps) => {
 
           <Flex gap="3" justify="end">
             <Dialog.Close>
-              <Button variant="outline" onClick={handleCancelClick}>
```

Comment:
> I think I encountered a bug, after following the happy path by checking the settings again it was presented as `regular` while the file on disk did indeed still include the correct proxy config. (A restart fixed it but we might want to look into it)

@Llandy3d there was a leftover logic from when Settings was still a page and the form state needed to be reset. This should fix the issue.

Reply:
It did 🥳